src/components/views/rooms/EventTile.tsx

```diff
@@ -267,7 +267,7 @@ interface IProps {
     showReactions?: boolean;
 
     // which layout to use
-    layout: Layout;
+    layout?: Layout;
```

Comment:
Won't this break something?

Reply:
what's the rationale here?

Reply:
A lot of the code will implicitely fallback to `Layout.Group`. I have made that prop mandatory but also added a default prop.

This missing prop on `EventTile` was highlighted by the JS=>TS migration